README.md

```diff
@@ -1,212 +1,217 @@
-
-```shell
-export BASE_HOSTNAME="TODO-provide-base-hostname"
-export KEYCLOAK_URL="TODO-your-keycloak-url"
-export KEYCLOAK_REALM=trusted-artifact-signer
-
-export TUF_URL=https://tuf.$BASE_HOSTNAME
-export OIDC_ISSUER_URL=$KEYCLOAK_URL/auth/realms/$KEYCLOAK_REALM
-export COSIGN_FULCIO_URL=https://fulcio.$BASE_HOSTNAME
-export COSIGN_REKOR_URL=https://rekor.$BASE_HOSTNAME
-export COSIGN_MIRROR=$TUF_URL
-export COSIGN_ROOT=$TUF_URL/root.json
-export COSIGN_OIDC_CLIENT_ID=$KEYCLOAK_REALM
-export COSIGN_OIDC_ISSUER=$OIDC_ISSUER_URL
-export COSIGN_CERTIFICATE_OIDC_ISSUER=$OIDC_ISSUER_URL
-export COSIGN_YES="true"
-export SIGSTORE_FULCIO_URL=$COSIGN_FULCIO_URL
-export SIGSTORE_OIDC_ISSUER=$COSIGN_OIDC_ISSUER
-export SIGSTORE_REKOR_URL=$COSIGN_REKOR_URL
-export REKOR_REKOR_SERVER=$COSIGN_REKOR_URL
-```
-
-2. Initialize the TUF roots
-
-```shell
-cosign initialize
-```
-
-Note: If you have used `cosign` previously, you may need to delete the `~/.sigstore` directory
-
-3. Sign the desired container
-
-```shell
-cosign sign -y <image>
-```
-
-Authenticate with the Keycloak instance using the desired credentials.
+## Prerequisites
 
-4. Verify the signed image
+* RHEL x86\_64 9.2 or greater.
+* Command-line access to the Ansible control node with a user that has `sudo` privileges.
+* Installation and configuration of Ansible on a control node to perform the automation.
+* Installation of the Ansible collections on the control node, unless installing from the Ansible Automation Hub.
+  > [!NOTE]
+  To install the required dependencies automatically, you can run the `ansible-galaxy install` command.
+* An OpenID Connect (OIDC) provider, such as [Keycloak](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/).
+* The ability to resolve the ingress host names, by using the Domain Name System (DNS) or the `/etc/hosts` file.
+* Optional.
+  Installation of the `podman` and [`cosign`](https://github.com/sigstore/cosign) binaries to verify that the RHTAS service is working as expected.
+
+## Deploying
+
+1. Create an `inventory` file with a single node under the `rhtas` group:
+   
+   ```
+   [rhtas]
+   123.123.123.123
+   ```
+
+2. Create a simple Ansible Playbook named `play.yml`, and replace `TODO` with your relevant information:
+   
+   ```yaml
+   - hosts: rhtas
+     vars:
+       base_hostname: TODO # e.g. example.com
+       # access credentials for registry.redhat.io (https://access.redhat.com/RegistryAuthentication)
+       tas_single_node_registry_username: TODO
+       tas_single_node_registry_password: TODO
+       tas_single_node_oidc_issuers:
+         - issuer: TODO # your OIDC provider (e.g. keycloak) URL
+           client_id: trusted-artifact-signer
+           url: TODO # your OIDC provider (e.g. keycloak) URL
+           type: email
+     tasks:
+       - name: Include TAS single node role
+         ansible.builtin.include_role:
+           name: redhat.artifact_signer.tas_single_node
```

Comment:
We should leave a comment to use just "tas_single_node" if running from a locally-cloned Git repo.

Reply:
What do you mean by "use just 'tas_single_node'"?   Does the user only need to update the `tas_single_node*` fields in this Playbook, if running from a locally cloned Git repo?  How does this differ from users who use this Playbook on Ansible Automation Hub?  Can you provide me examples of what you mean.

Reply:
So if you're using from a local checkout you'll need to use `name: tas_single_node` here. If you install this collection from AAH via `ansible-galaxy install`, you'll need to use `name: redhat.artifact_signer.tas_single_node`, that's the only difference. Does that make sense?

Reply:
Yes, that makes sense.  Thank you for clarifying!